systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -10,64 +10,57 @@
 % the answer).
 %
 
-if model.use_new_kinsol
-  checkDirty(model);  
-  if nargin<6
+checkDirty(model);
+
+% method signature transition
+if nargin > 3
+  options_copy = options;
+end
+warn_signature_changed = false;
+if nargin == 5
+  warn_signature_changed = true;
+  options = struct();
+  options.compute_gradients = v;
```

Comment:
compute_gradients is a confusing name for this.  we always compute first derivatives, and this is only for second derivatives.. correct?